server/service/src/programs/encounter/insert.rs

```diff
@@ -68,8 +73,11 @@ pub fn insert_encounter(
             // Updating the document will trigger an update in the patient (names) table
             let result = service_provider
                 .document_service
-                .update_document(ctx, doc)
+                .update_document(ctx, doc, &allowed_docs)
                 .map_err(|err| match err {
+                    DocumentInsertError::NotAllowedToMutDocument => {
```

Comment:
:nit I would prefer `NotAllowedToMutateDocument` as I don't think that `Mut` is a generally accepted abbreviation. Makes sense in a rust context.. but beyond that? 🤷

Reply:
thanks I changed it, Mut is fine for Rust code but the Mut was also used in graphql. I renamed all Mut to Mutate.